### PR TITLE
set nlu fallback confidence to 1 - intent confidence

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- fixes https://github.com/RasaHQ/rasa/issues/6685

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/components.mdx [hunk 1]
```diff
@@ -1572,7 +1572,7 @@ See [starspace paper](https://arxiv.org/abs/1709.03856) for details.
 * **Short**
 
   Classifies a message with the intent `nlu_fallback` if the NLU intent classification
-  scores are ambiguous.
+  scores are ambiguous. The confidence is set to 1 - top intent confidence.
```
Comment (wochinge) [inline docs/docs/components.mdx:1575]:
```suggestion
  scores are ambiguous. The confidence is set to `1 - top intent` confidence.
```

File: changelog/7278.improvement.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The confidence of the [`FallbackClassifier`](components.mdx#fallbackclassifier)
+predictions is set to 1 - top intent confidence.
```
Comment (wochinge) [inline changelog/7278.improvement.md:2]:
```suggestion
predictions is set to `1 - top intent` confidence.
```

**Review by melindaloubser1 — APPROVED** [review]

File: docs/docs/components.mdx (hunk 1 above)
Comment (Ghostvv) [reply docs/docs/components.mdx:1575]: then it should be `1 - top intent confidence` but that's too much for ``, no?

Comment (wochinge) [reply docs/docs/components.mdx]: I'd put it all in '`' but up to you

**Pull request merged**